src/lib/style-properties.html

```diff
@@ -147,10 +152,11 @@
         var parts = property.split(';');
         for (var i=0, p, m; i<parts.length; i++) {
           if ((p = parts[i])) {
-            m = p.match(this.rx.MIXIN_MATCH);
+            this.rx.MIXIN_MATCH.lastIndex = 0;
+            m = this.rx.MIXIN_MATCH.exec(p);
             if (m) {
               p = this.valueForProperty(props[m[1]], props);
-            } else {
+            } else if (!settings.useNativeCSSProperties) {
```

Comment:
Why do we need the check for native here? Shouldn't we never come here if using native?